src/db_options.rs

```diff
@@ -2188,6 +2189,24 @@ impl Options {
         }
     }
 
+    /// if not zero, dump rocksdb.stats to RocksDB every stats_persist_period_sec
+    ///
+    /// Default: 0
+    /// 
+    /// # Examples
+    ///
+    /// ```
+    /// use rocksdb::Options;
+    ///
+    /// let mut opts = Options::default();
+    /// opts.set_stats_persist_period_sec(0);
+    /// ```
+    pub fn set_stats_persist_period_sec(&mut self, period: c_uint) {
+        unsafe {
+            ffi::rocksdb_options_set_stats_persist_period_sec(self.inner, period);
```

Comment:
As far as I can see, the `rocksdb_options_set_stats_persist_period_sec` function isn't present in the rocksdb C-api. Looks like it should be exported.

Reply:
@DarkEld3r I will export

Reply:
@DarkEld3r https://github.com/facebook/rocksdb/pull/7168